libs/components/flyout/src/lib/modules/flyout/flyout-media-query.service.ts

```diff
@@ -1,26 +1,23 @@
 import { Injectable } from '@angular/core';
 import { SkyMediaBreakpoints, SkyMediaQueryListener } from '@skyux/core';
 
-import { BehaviorSubject, Subscription } from 'rxjs';
+import { ReplaySubject, Subscription } from 'rxjs';
 
 /**
  * @internal
  */
 @Injectable()
 export class SkyFlyoutMediaQueryService {
-  public get current(): SkyMediaBreakpoints {
+  public get current(): SkyMediaBreakpoints | undefined {
     return this.#_current;
   }
 
-  #currentSubject: BehaviorSubject<SkyMediaBreakpoints>;
+  #currentSubject: ReplaySubject<SkyMediaBreakpoints>;
 
-  #_current = SkyMediaBreakpoints.xs;
+  #_current: SkyMediaBreakpoints | undefined;
```

Comment:
It looks like we're still referencing `this.#_current` outside the getter. Since that's the case, it should be renamed `#current`.

Reply:
We actually can just get rid of that private variable and modify and use the public one. Done.